package rpcprotocol;

message RpcMessage {
  message Detail {
    extensions 1 to 18999;  // all extensions must themselves be proto messages
  }
  required uint32 rpc_id = 1;
  optional bytes method = 2;
  required Detail detail = 3;
};

